src/measurements/addons.js

```diff
@@ -0,0 +1,14 @@
+import { AddonManager } from 'resource://gre/modules/AddonManager.jsm';
+import BaseMeasurement from '../lib/measurement';
+
+export default class Addons extends BaseMeasurement {
+  measure(resolve) {
+    AddonManager.getAllAddons(addons => {
+      resolve(
+        addons
+          .filter(addon => addon.isActive)
+          .map(addon => addon.id)
```

Comment:
Interesting fact: this includes all hidden built-in add-ons as well.

For example, I only have the Pulse add-on installed, but the ping includes ~13 add-ons (including Pocket, etc).

![add-ons_manager_and_include_list_of_enabled_add-ons_in_reports__closes__24___by_chuckharmston_ _pull_request__67_ _mozilla_pulse_and_1____npm_bin__jpm_run_--addon-dir_build_-b_nightly__firefox-bin__and_index_js_ _build_webextension_survey_](https://cloud.githubusercontent.com/assets/557895/22610035/5eb2770a-ea18-11e6-8456-5d554b7f928d.png)


![browser_console_and_builds_user_sentiment_survey__closes__10__closes__18___by_chuckharmston_ _pull_request__82_ _mozilla_pulse](https://cloud.githubusercontent.com/assets/557895/22610021/4f933ec6-ea18-11e6-8b6d-dab77c95b89d.png)